core/common/app.php

```diff
@@ -61,6 +62,7 @@ public function init_components() {
 		}
 
 		$this->add_component( 'connect', new Connect() );
+		$this->add_component( 'devTools', new Devtools() );
```

Comment:
Currently - move it to `/modules` as a separated module.

Reply:
Handled.